pgscatalog.core/src/pgscatalog/core/__init__.py

```diff
@@ -14,56 +15,20 @@
     TargetVariants,
     TargetVariant,
     TargetType,
-    BasePGSException,
-    MatchError,
-    DuplicateMatchError,
-    MatchRateError,
-    ZeroMatchesError,
-    MatchValueError,
-    CombineError,
-    BuildError,
-    ScoreFormatError,
-    EffectTypeError,
-    CatalogError,
-    ScoreDownloadError,
-    ScoreChecksumError,
-    QueryError,
-    InvalidAccessionError,
-    SamplesheetError,
-    GenomesNotFound,
-    SamplesheetFormatError,
```

Comment:
Removed these exceptions from public API because they're only used internally by the CLI applications